src/components/footer.vue

```diff
@@ -9,12 +9,12 @@
                     Our main goal is to provide best possible freemium solution without 3rd parties for accommodation in The Netherlands. </p>
                 </div>
                 <div class="col-md-2 col-sm-2 text-center">
-                    <h3>Quick  LInks</h3>
+                    <h3>Quick  Links</h3>
                     <ul class="footer-links text-center">
-                        <li><a href="#."><router-link to="/">Home</router-link></a></li>
-                        <li><a href="#."><router-link to="">Search</router-link></a></li>
-                        <li><a href="#."><router-link to="register">Registration</router-link></a></li>
-                        <li><a href="#."><router-link to="login">LogIn</router-link></a></li>
+                        <li><a><router-link to="/">Home</router-link></a></li>
```

Comment:
as I said before <a> tag is useless here. now it will render <a><a href="/">Home</a></a> outer <a> provides nothing.